.eslintrc

```diff
@@ -230,6 +230,19 @@
         "process": false,
         "require": false
       }
+    },
+    {
+      "files": ["**/*.extern.js"],
+      "rules": {
+        "no-var": "off",
+        "no-undef": "off",
+        "no-unused-vars": "off",
+        "prefer-const": "off",
+        "require-jsdoc": "off",
+        "jsdoc/check-tag-names": "off",
+        "local/closure-type-primitives": "off",
+        "google-camelcase/google-camelcase": "off"
```

Comment:
For consistency with the rest of this file, use `0` instead of `"off"`.

Also, I wish your initial approach (of enabling just the `prettier/prettier` rule for extern files) had worked. With the explicit approach here, it's possible that a future change to an extern file can trigger arbitrary formatting rules that will then have to be explicitly disabled. Not sure there's a way around this though.

Reply:
I poked around a little and found that you can automatically exclude all rules that conflict with prettier by adding an `extends` section to your override config and listing `eslint-plugin-prettier` in it (or simply, `prettier`):

```yaml
    {
      "files": ["**/*.extern.js"],
      "extends": ["prettier"],
      "rules": {
        "prettier/prettier": 2
      }
    }
```

With this, I suspect the list of conflicting rules that need to be explicitly disabled will be considerably shorter.

Reply:
> For consistency with the rest of this file, use 0 instead of "off".

Done.

> With this, I suspect the list of conflicting rules that need to be explicitly disabled will be considerably shorter.

I'll ping you, I'm not sure how this would work

Reply:
> > With this, I suspect the list of conflicting rules that need to be explicitly disabled will be considerably shorter.
> 
> I'll ping you, I'm not sure how this would work

Hmm. I dug in some more, and it turns out we're already using that plugin in the main section of `.eslintrc`, and therefore, we're already auto-excluding default `eslint` rules that may conflict with `prettier`.

I guess the only option left is to explicitly disable specific conflicting rules for extern files, as you are currently doing.

Thanks for looking into this!